src/main/java/me/earth/earthhack/impl/modules/render/popchams/ListenerTotemPop.java

```diff
@@ -50,7 +51,8 @@ public void invoke(TotemPopEvent event) {
                         event.getEntity().rotationPitch,
                         event.getEntity().posX,
                         event.getEntity().posY,
-                        event.getEntity().posZ
+                        event.getEntity().posZ,
+                        ((AbstractClientPlayer)event.getEntity()).getSkinType().equals("slim")
```

Comment:
Can please have an instanceof check? Just in case someone implements EntityPlayer themselves?

Reply:
> They look awesome btw!

thanks!